src/In.ProjectEKA.DefaultHip/Resources/dignosticReportWithMediaDicomFileAsUrl.json

```diff
@@ -0,0 +1,55 @@
+{
+  "resourceType": "Bundle",
+  "id": "11123232324",
+  "type": "collection",
+  "entry": [{
+    "fullUrl": "123efdfzxa87",
```

Comment:
try to use unique uuids for "resources". 
and for full url - use "urn:uuid:<uuid>". not the fullURL must be compliant with the "id" of the resource. 
you can use https://www.uuidgenerator.net/ to generate unique uuids